packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -41,17 +42,19 @@ export const renderProgramCard = ({
   duration = 'PT30M',
   durationLabel = 'Duration',
   startTime = 1566914061212,
+  displaySummary = boolean('show summary', true),
 }) => {
   const { text, articlePath, longText, dir, locale, timezone } = TEXT_VARIANTS[
     service
   ];
+
   return (
     <ProgramCard
       service={service}
       script={dir === 'rtl' ? arabic : latin}
       dir={dir}
       brandTitle={text}
-      summary={longText}
+      summary={displaySummary ? longText : null}
```

Comment:
```suggestion
      summary={displaySummary && longText}
```

Reply:
Opted to use a ternary operator as opposed to `&&` because the latter introduces a prop type warning. I thought it makes sense to set summary to `null` because that's what is expected when there is no summary

Reply:
Ok, sounds good to me 👍